torchbenchmark/models/sam/install.py

```diff
@@ -0,0 +1,23 @@
+import os
+import subprocess
+import sys
+
+def pip_install_requirements():
+    subprocess.check_call([sys.executable, '-m', 'pip', 'install', '-q', '-r', 'requirements.txt'])
+
+def download_checkpoint():
+    subprocess.check_call(['wget', '-P', '.data', 'https://dl.fbaipublicfiles.com/segment_anything/sam_vit_h_4b8939.pth'])
+
+def download_data():
+    subprocess.check_call(['wget', '-P', '.data', 'https://github.com/facebookresearch/segment-anything/blob/main/notebooks/images/truck.jpg?raw=true'])
+
+if __name__ == '__main__':
+    pip_install_requirements()
+
+    # Create .data folder in the script's directory
+    data_folder = os.path.join(os.path.dirname(os.path.abspath(__file__)), '.data')
```

Comment:
@xuzhao9 is this kind of folder what you had in mind? Still seeing an annoying error where this is failing in CI but passing locally

Reply:
NVM fixed this